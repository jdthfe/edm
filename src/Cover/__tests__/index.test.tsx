import { render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import React from 'react';
import { Cover } from '@src/index';

describe('Cover', () => {
    // Supplement uncovered test
    it('trigger event correctly', () => {
        expect(true).toBe(true);
    });

    it('Cover show successful', () => {
        expect(renderToJson(render(<Cover />))).toMatchSnapshot();
    });

    it('Cover.Transition show successful', () => {
        expect(renderToJson(render(<Cover.Transition />))).toMatchSnapshot();
    });
});
