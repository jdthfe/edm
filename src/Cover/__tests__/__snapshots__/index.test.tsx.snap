// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover Cover show successful 1`] = `
<div
  class="edm-Cover"
/>
`;

exports[`Cover Cover.Transition show successful 1`] = `null`;
